src/GraphBLAS-sharp/Predefined/Boolean.fs

```diff
@@ -0,0 +1,15 @@
+namespace GraphBLAS.FSharp.Predefined
+
+open GraphBLAS.FSharp
+
+module BooleanMonoid =
+    let any: Monoid<bool> = {
+        Zero = false
+        Append = BinaryOp <@ ( || ) @>
+    }
+
+module BooleanSemiring =
+    let anyAll: Semiring<bool> = {
```

Comment:
Huh... I'm confused. Here we have semiring with one type parameter, but in some files, we use semiring with three type parameters.